package.json

```diff
@@ -8,9 +8,9 @@
     "test": "test"
   },
   "scripts": {
+    "build": "gulp build",
     "test": "gulp test",
-    "preinstall": "npm install -g typings",
-    "postinstall": "typings install"
+    "contrib": "typings install && npm run build"
```

Comment:
Since this now runs build, it would require typescript as well.

Also, if a contributor has to manually run `npm run contrib` it's not any different than them manually running `typings install`. 

The `npm run build` on the end seems unnecessary since there are already separate build and test scripts.